server/engine/src/main/resources/resourceBundles/messages_en.properties

```diff
@@ -132,6 +132,8 @@ rewriteFileOperation.notOpened=REWRITE statement can only reference a file opene
 deleteFileOperation.notOpened=DELETE statement can only reference a file opened for I-O
 startFileOperation.notOpened=START statement can only reference a file opened for INPUT or I-O
 compilerDirective.deprecatedDirectiveUse=%s is a deprecated compiler directive
+compilerDirective.warning.deprecatedDirectiveUse=The "%s" compiler option was specified, but is not supported.
+compilerDirective.info.deprecatedDirectiveUse=The "%s" option is no longer supported.
```

Comment:
@zeibura Please review
IBM Cobol compiler gives below information
`IGYOS4013-I   The "SIZE" option is no longer supported.`

Reply:
this is fine